packages/ncids-twig/components/nci-card/nci-card.twig

```diff
@@ -0,0 +1,24 @@
+<div id="nci-card" class="grid-container">
+	{% if float %}
+		<div class="usa-prose">
```

Comment:
The twig file should **only** be the card itself. All the content that belongs to the story to help with testing needs to live in the storybook files.